bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp

```diff
@@ -806,6 +806,7 @@ TEST_F(BcStTest, validatePeriodicSourceReplacement) {
   }
   const auto& sources_ = getSourceSelector().getActualSources();
   ASSERT_EQ(sources_.size(), 3);
+  ASSERT_EQ(2, getSourceSelector().metrics_.periodic_source_replacement_.Get().Get());
```

Comment:
You should check that all other counters are 0, to be sure.
I would create a helper function for that - check expected vs actual. All defaults are set to 0.I think it will be a good habit to check that all counters are 0 in all tests where we do not expect any replacment (we can add that in the future or you can add it here).